.changeset/chilled-terms-buy.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/table': minor
```

Comment:
This Table change is breaking since it will substantially affect page layouts.

We should also make sure https://github.com/mongodb/leafygreen-ui/pull/637 is merged in before this PR.

Reply:
👍